src/Sylius/Behat/Context/Api/Admin/ManagingProductsContext.php

```diff
@@ -548,20 +555,35 @@ private function getFieldValueOfFirstProduct(array $product, string $field): ?st
         }
 
         if ($field === 'name') {
-            return $product['translations']['en_US']['name'];
+            return $product['translations'][$this->getAdminLocaleCode()]['name'];
         }
 
         return null;
     }
 
+    private function hasProductImage(Response $response, ProductInterface $product): bool
+    {
+        $productFromResponse = $this->responseChecker->getResponseContent($response);
+
+        if (
+            isset($productFromResponse['images']) &&
+            isset($productFromResponse['images'][0]) &&
```

Comment:
```suggestion
            isset($productFromResponse['images'][0]) &&
```
Should work exactly same